iml-services/iml-action-runner/src/local_actions.rs

```diff
@@ -125,9 +124,6 @@ pub async fn handle_local_action(
                         await_next_session(fqdn, last_session, wait_secs, sessions)
                     })
                 }
-                "get_fqdn_by_id" => {
```

Comment:
Why is this change needed?

Reply:
I believe we discussed removing it a while back since it was moved to graphql.